src/main/resources/liquibase/crop_changelog/15_4_1.xml

```diff
@@ -0,0 +1,178 @@
+		<sql dbms="mysql">
+			DELETE FROM projectprop WHERE variable_id=8251 AND project_id IN (SELECT project_id FROM project WHERE `name` LIKE '%Nursery Template%');
+		</sql>
+	</changeSet>
+
+	<changeSet id="v15.4.1-6" author="corina">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">
+				SELECT COUNT(*) FROM cvterm where cvterm_id=4021; 
+			</sqlCheck>
+		</preConditions>
+		<comment>Change method of variables with same property-method-scale triplet. IBP-3800</comment>
+		<sql dbms="mysql">
+			SET FOREIGN_KEY_CHECKS=0;
+			
+			REPLACE INTO `cvterm`(`cvterm_id`,`cv_id`,`name`,`definition`,`dbxref_id`,`is_obsolete`,`is_relationshiptype`) VALUES (4021,1020,'Applied at plot level','Applied at plot level',NULL,0,0);
+
+			UPDATE cvterm_relationship SET object_id=4021 WHERE type_id=1210 AND subject_id IN (8258,8259,8253);
+			
+			SET FOREIGN_KEY_CHECKS=1;
+		</sql>
+	</changeSet>
+
+       <changeSet id="v15.4.1-7" author="corina">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">SELECT count(*) > 1 from cvterm WHERE `name` IN ('EW_DW_kgPlot','EW_FW_gPlot','EW_FW_kgPlot');
+			</sqlCheck>
+		</preConditions>
+		<comment>
+			Apply correction on methods of variables EW_DW_kgPlot, EW_FW_gPlot, EW_FW_kgPlot. IBP-3930.
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			SET FOREIGN_KEY_CHECKS=0;
+			
+			UPDATE cvterm c1, cvterm_relationship c2
+			SET c2.object_id=61390,
+			c1.definition=REPLACE(definition,'Field weight','Dry weight')
+			WHERE c1.`name`='EW_DW_kgPlot' AND c1.cvterm_id=51574
+			AND c1.cvterm_id=c2.subject_id
+			AND c2.type_id=1210;
+
+			UPDATE cvterm c1, cvterm_relationship c2
+			SET c2.object_id=61398,
+			c1.definition=REPLACE(definition,'Dry weight','Field weight')
+			WHERE c1.`name`='EW_FW_gPlot' AND c1.cvterm_id=51675
+			AND c1.cvterm_id=c2.subject_id
+			AND c2.type_id=1210;
+			
+			UPDATE cvterm c1, cvterm_relationship c2
+			SET c2.object_id=61398,
+			c1.definition=REPLACE(definition,'Dry weight','Field weight')
+			WHERE c1.`name`='EW_FW_kgPlot' AND c1.cvterm_id=50854
+			AND c1.cvterm_id=c2.subject_id
+			AND c2.type_id=1210;
+
+			SET FOREIGN_KEY_CHECKS=1;
+		</sql>
+	</changeSet>
+
+	<changeSet id="v15.4.1-8" author="corina">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">
+				SELECT COUNT(*) from cvterm where cvterm_id=8265 and cv_id=1040; 
+			</sqlCheck>
+		</preConditions>
+		<comment>Update any usage of var 8265 (FertNTotal_kg_ha) to 8260 (NFert_kg_ha), then delete 8265 from ontology as it duplicates 8260. IBP-3800</comment>
+		<sql dbms="mysql">
+			SET FOREIGN_KEY_CHECKS=0;
+			
+			UPDATE projectprop SET variable_id=8260,alias="NFert_kg_ha" WHERE variable_id=8265;
+			UPDATE stockprop SET type_id=8260 WHERE type_id=8265;
+            UPDATE nd_experimentprop SET type_id=8260 WHERE type_id=8265;
+            UPDATE nd_geolocationprop SET type_id=8260 WHERE type_id=8265;
```

Comment:
We should also update phenotype table:
` UPDATE phenotype SET observable_id=8260 WHERE observable_id=8265;`

And also delete from `cvtermsynonym`?
`DELETE FROM cvtermsynonym where cvterm_id=8265;`